packages/core-data/src/entities.js

```diff
@@ -12,29 +17,49 @@ import { apiFetch, select } from './controls';
 export const DEFAULT_ENTITY_KEY = 'id';
 
 export const defaultEntities = [
-	{ name: 'site', kind: 'root', baseURL: '/wp/v2/settings' },
-	{ name: 'postType', kind: 'root', key: 'slug', baseURL: '/wp/v2/types' },
+	{
+		label: __( 'Site' ),
```

Comment:
I had to introduce this "label" property and also a dependency towards i18n package.